src/js/themes/hpe.js

```diff
@@ -833,6 +833,59 @@ export const hpe = deepFreeze({
       color: 'text-strong',
     },
   },
+  notification: {
+    actions: {
+      color: { light: 'text', dark: 'text-strong' },
+    },
+    direction: 'row',
+    container: {
+      round: 'xsmall',
+    },
+    global: {
+      container: {
+        round: 'none',
+      },
+    },
+    critical: {
+      background: 'validation-critical',
+      toast: {
```

Comment:
I think in latest grommet stable we now have a defined color for notification[status]global.background -- which we might need to override to ensure out validation colors are coming through.

Can you double check that?

Reply:
yes Ill check now

Reply:
yes the critical is coming through so we don't need that just the rest

Reply:
looks like we dont need `toast` background colors either